deploy/docker-compose.yml

```diff
@@ -11,14 +11,14 @@ services:
     restart: unless-stopped
 
   python:
-    image: gcr.io/linksight-208514/python:latest
+    image: gcr.io/linksight-208514/python:${BUILD_ID}
```

Comment:
Is there a reason some of these are braced and some are not? Would be nice if we had more consistency.

Reply:
https://docs.docker.com/compose/environment-variables/#configure-compose-using-environment-variables